src/components/categoriesScreen/Categories.tsx

```diff
@@ -0,0 +1,194 @@
+import React, { useEffect, useState } from "react";
+import {
+  View,
+  FlatList,
+  Text,
+  Image,
+  StyleSheet,
+  TouchableOpacity,
+  ActivityIndicator,
+} from "react-native";
+import { Category, FetchCategoriesResponse } from "../../types/types";
+import businessManagerService from "../../services";
+import { dimensions } from "../../styles/branding";
+import { primary } from "../../styles/styleGuide";
+import { generalStyles } from "../../styles/generalStyleSheet";
+import { CategoryBackground } from "../../utils/svg";
+
+const HomeScreenCategories = () => {
+  const [categories, onChangeCategories] = useState<
+    FetchCategoriesResponse | undefined
+  >(undefined);
+  const [isLoading, setLoading] = useState<boolean>(false);
+
+  interface FlatListProps {
+    item: Category;
+    index: number;
+  }
+
+  const renderItem = ({ item, index }: FlatListProps) => {
+    const isItemPositionEven = index % 2 === 0;
+    return (
+      <TouchableOpacity
+        activeOpacity={0.8}
+        style={{ marginLeft: 50, marginRight: 50 }}
+      >
+        <View
+          style={[
+            styles.container,
+            {
+              flexDirection: isItemPositionEven ? "row" : "row-reverse",
+              elevation: 10,
+            },
+          ]}
+        >
+          <View
+            style={[
+              styles.textItemViewStyle,
+              {
+                marginLeft: isItemPositionEven ? 50 : 0,
```

Comment:
you can creata a const 
const itemPosition =           {
                  marginLeft: isItemPositionEven ? 50 : 0,
                  marginRight: isItemPositionEven ? 0 : 50,
                },

and use it